source/includes/inkling-guide/_mental-models.html.md

```diff
@@ -1,8 +1,10 @@
-## Mental Models
+# Mental Models
 
-Mental models are a way of organizing concepts so that you can structure your data flow. Concepts are the ideas that you want to teach your BRAIN.
+Mental models are a way of organizing concepts so that you can structure a solution to your AI problem as a workflow. Concepts are the ideas that you want to teach your BRAIN. When you create your concepts, you're going to start with the final concept that you want to teach your BRAIN and support it with other fact and strategy concepts. These additional concepts appear upstream of your main concept because the information output from those concepts helps the Bonsai AI Engine teach your main concept to your BRAIN. Mental Models enable you to determine and plan the concepts that are taught to your BRAIN.
```

Comment:
This might just be a bug to open but instead of saying "fact and strategy"--words bonsai invented we should probably switch to "feature and skill" which are words found in the Machine Learning literature